renovate.json

```diff
@@ -57,10 +58,12 @@
     {
       "fileMatch": [
         ".github/workflows/ci.yaml",
-        ".github/workflows/generated_files_sync.yaml"
+        ".github/workflows/generated_files_sync.yaml",
+        ".github/workflows/hermetic_library_generation.yaml"
       ],
       "matchStrings": [
-        "library_generation_image_tag\\s*:\\s*(?<currentValue>.*?)\\n"
+        "library_generation_image_tag\\s*:\\s*(?<currentValue>.*?)\\n",
+        "uses: googleapis/sdk-platform-java/.github/scripts@v(?<currentValue>.+?)\\n"
```

Comment:
This is great! All the generator updates are grouped together! I assume the renovate config is already tested locally?

Reply:
Yes, I tested the renovate using:
```
export LOG_LEVEL=debug && npx --yes --package renovate -- renovate --platform=local
```